src/coreComponents/mesh/generators/HexCellBlockManager.cpp

```diff
@@ -80,50 +80,57 @@ static constexpr HexVertexIndex edgeAdjacentFacet[12][2]{
 };
 
 
-// Structure for face computation
-// TODO Optimize: replace (v0, v1, v2, f) by (v0*nbNodes + v1, v2, f)
+/* Structure for face computation
+ * The table stores 3 sorted LocalVertexIndex v0 < v1 < v2
+ * and f the LocalFaceIndex that identifies unambiguously a face in a cell
+ *  v3 =  Hex::nbFacets * idCell + f_hex
+ * where f_hex is the HexFaceIndex
+ * 
+ * TODO Optimize: replace (v0, v1, v2) by (v0*nbNodes + v1, v2)
+ */
 struct FaceInfo{
-  localIndex v[4];
-};
 
-struct {
-  bool operator()( FaceInfo const & a, FaceInfo const & b) const {
-    if(a.v[0] != b.v[0]) return a.v[0] < b.v[0];
-    if(a.v[1] != b.v[1]) return a.v[1] < b.v[1];
-    if(a.v[2] != b.v[2]) return a.v[2] < b.v[2];
+  /* Compare the indices of the 3 sorted vertex indices
+  */
+  bool operator<( FaceInfo const & right) const 
```

Comment:
This is not really what I had in mind.
I was refering to 
```c++
  std::sort( allFaces.begin(), allFaces.end(), [](FaceInfo const & left, FaceInfo const & right) -> true {
    if(left.v[0] != right.v[0]) return left.v[0] < right.v[0];
    if(left.v[1] != right.v[1]) return left.v[1] < right.v[1];
    if(left.v[2] != right.v[2]) return left.v[2] < right.v[2];
    return false;
} );
```
Because you use this operator just once.
Also, I've always been uncomfortable with those comparator which do not take into account all the parameters. But it's maybe personal.

Reply:
I know that is not what you had in min. That is an overloading of the regular C++  operators.